src/ui/PrintableUri/PrintableUri.ts

```diff
@@ -165,6 +189,29 @@ export class PrintableUri extends Component {
     ).el;
   }
 
+  private buildEllipsis(action: (e: Event) => void) {
```

Comment:
In a nutshell: This builds a `listitem` element containing a `button`.

Making the parent `listitem` button clickable isn't correctly conveyed to accessibility tools, this is why other `listitem` elements in this `list` have a `link` element inside them.